openpype/hosts/maya/api/lib.py

```diff
@@ -4100,15 +4100,24 @@ def create_rig_animation_instance(
     """
     if options is None:
         options = {}
-
+    name = context["representation"]["name"]
     output = next((node for node in nodes if
                    node.endswith("out_SET")), None)
     controls = next((node for node in nodes if
                      node.endswith("controls_SET")), None)
-
-    assert output, "No out_SET in rig, this is a bug."
-    assert controls, "No controls_SET in rig, this is a bug."
-
+    if name != "fbx":
+        assert output, "No out_SET in rig, this is a bug."
+        assert controls, "No controls_SET in rig, this is a bug."
+
+    anim_skeleton = next((node for node in nodes if
+                          node.endswith("skeletonAnim_SET")), None)
+    if not anim_skeleton:
+        log.debug("No skeletonAnim_SET in rig")
+
+    skeleton_mesh = next((node for node in nodes if
+                          node.endswith("skeletonMesh_SET")), None)
+    if not skeleton_mesh:
+        log.debug("No skeletonMesh_SET in rig")
```

Comment:
So the sets above do not exist for an `fbx` - but these do?
Also, do we care enough to be debug logging these?

Reply:
if we dont add this, i guess the fbx import would error out to load just the skeletonMesh set.